images/public-log-vendor-matcher/README.md

```diff
@@ -0,0 +1,44 @@
+# Public log vendor matcher
+
+A Postgres-based k8s-infra data pipeline that produces BigQuery tables for reviewing Kubernetes Public artifact traffic in DataStudio.
```

Comment:
Need more of a description of the set of steps this takes.

(totally making this up)
1. scrapes logs from $place
2. anonymizes the following fields:
  - $foo by doing $x
  - $bar by doing $y
3. write the resulting anonymized data to $other_place
4. aggregates the data by doing $z and writes results to $place_that_could_be_promoted

the $promoted_place houses A B and C and is intended to used by E to display F

Reply:
so e.g. describe the pg-init.d stuff here, and what gets invokled when, and how the app directory gets invoked, etc.

Reply:
pg-init.d still isn't described in sufficient detail, I'm looking something on the level of

- when the container starts
- something launches postgres with the password baked in the container
- something kicks off pg-init.d
- pg-init.d then does the following
  - runs through all files in /foo/bar in alphabetical order
    - if the file is executable, runs it
    - if the file is sql, executes it against the local postgres instance
 - if all files successfully executed, runs /app/foo/bar

etc. (I made all that up, maybe that's not how it actually works)

Reply:
Updated in https://github.com/kubernetes/k8s.io/commit/b767950c205f03cc8345447d31032a6cd2f10b36

Reply:
https://github.com/kubernetes/k8s.io/pull/2710/files#r718612992 still isn't addressed AFAICT, I don't see any references to pg-init.d in this README